extension/js/common/core/crypto/key.ts

```diff
@@ -100,6 +100,8 @@ export type PrvPacket = (OpenPGP.packet.SecretKey | OpenPGP.packet.SecretSubkey)
 
 export class UnexpectedKeyTypeError extends Error { }
 
+export type ArmoredKeyWithEmailsAndId = { id: string, emails: string[], armored: string };
```

Comment:
We can extend this type (interface) from `KeyIdentity` and set `family` too, so we can later use `KeyUtil.identityEquals` in `ComposeMyPubkeyModule.reevaluateShouldAttachOrNot` instead of comparing fingerprints.